src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -70,11 +70,106 @@ Continue to [Step 2. Create an AWS endpoint](#step-2-create-an-aws-endpoint).
 1. Select the **PrivateLink** tab. PrivateLink connections that have already been configured are shown as a private endpoint allowlist.
 1. To add a new private connection, click **Add Connection** to open the connection dialog.
 
-Continue to [Step 2. Create an AWS endpoint](#step-2-create-an-aws-endpoint).
+Continue to [Step 3. Create an AWS endpoint](#step-3-create-an-aws-endpoint).
 
 </section>
 
-## Step 2. Create an AWS endpoint
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+## Step 2. (Optional) Configure private endpoint trusted owners
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/limited-access.md %}
+{{site.data.alerts.end}}
+
+Optionally, you can restrict the AWS accounts that can connect to your cluster privately using private endpoints. During Limited Access, to configure trusted owners, you must use the [CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}) or [Terraform Provider](https://github.com/cockroachdb/terraform-provider-cockroach/).
+
+Keep the following in mind:
+- This feature can be enabled only on clusters created after your organization is enrolled in the Limited Access.
```

Comment:
nit: you might want a new line after "Keep the following in mind:". It looks like the points are not properly formatted. 
<img width="1470" alt="image" src="https://github.com/cockroachdb/docs/assets/83607639/873ab438-7560-45aa-89ba-80fdc80cdbf2">